build_tools/github_actions/lint_version.sh

```diff
@@ -21,8 +21,8 @@
 VERSION_H="stablehlo/dialect/Version.h"
 set_version_var() {
   # getCurrentVersion() { Version(0, X, Y); }
-  VERSION_STR=$(cat $VERSION_H | grep getCurrentVersion -A1 | grep -o 'Version([0-9], .*)')
-  REGEX="Version\(([0-9]+), ([0-9]+), ([0-9]+)\)"
+  VERSION_STR=$(cat $VERSION_H | grep getCurrentVersion -A1 | grep -o 'Version(.*[0-9])')
+  REGEX="Version\(/\*.*=\*/([0-9]+), /\*.*=\*/([0-9]+), /\*.*=\*/[^0-9]*([0-9]+)\)"
```

Comment:
@GleasonK This is the additional change we have to incorporate some of the readability comments. PTAL